tracer/src/Datadog.Trace/Telemetry/Collectors/ConfigurationTelemetryCollector.cs

```diff
@@ -128,6 +128,7 @@ public ICollection<TelemetryValue> GetConfigurationData()
                 new(name: "tracer_instance_count", value: _tracerInstanceCount),
                 new(name: "security_enabled", value: _securitySettings?.Enabled),
                 new(name: "security_blocking_enabled", value: _securitySettings?.BlockingEnabled),
+                new(name: "environment_fulltrust_appdomain", value: AppDomain.CurrentDomain.IsFullyTrusted),
```

Comment:
more a comment for Andrew's PR but that makes me realize: Why don't we use the constants defined in `ConfigTelemetryData` here?

Reply:
Because of too much refactoring and me missing it 😂 I'll do that 👍